frontend/src/container/MetricsApplication/Tabs/Overview.tsx

```diff
@@ -92,6 +94,16 @@ function Application({ getWidget }: DashboardProps): JSX.Element {
 		}
 	};
 
+	const onDragSelect = useCallback(
+		(start: number, end: number) => {
+			const startTimestamp = Math.trunc(start);
```

Comment:
@volodfast startTimestamp is in `milliseconds` and if we supply in `nanoseconds` we don't need to change the internals of the global component